v-next/hardhat-node-test-reporter/integration-tests/index.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  isDirectory,
+  readdir,
+  getAllFilesMatching,
+  readUtf8File,
+} from "@nomicfoundation/hardhat-utils/fs";
+import { run } from "node:test";
+
+import reporter from "../src/reporter.js";
+import { diff } from "jest-diff";
+const SHOW_OUTPUT = process.argv.includes("--show-output");
+
+for (let entry of await readdir(import.meta.dirname + "/fixture-tests")) {
```

Comment:
I think the variable "entry" can be a const instead of let because it's not modified

Reply:
oh, good catch! the linter wasn't enabled for these tests.